### PR TITLE
Adding yarn, upgrading everything to latest except phantom.

### PR DESCRIPTION
**Pull request by choshun**
Phantom breaks at 2.x with error `UnhandledPromiseRejectionWarning: Unhandled promise rejection phantom 2`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,12 +38,16 @@
...
+    "image-size": "^0.5.1",
+    "imagely": "imagely-v0.1.12.tgz",
+    "lodash": "^4.17.4",
+    "phantom": "1.x",
+    "phantomjs": "^2.1.7",
+    "request-promise": "^4.1.1",
+    "yargs": "^7.0.1"
+  },
+  "devDependencies": {
+    "yarn": "^0.21.3"
```
Comment (mpetrovich) [inline package.json:51]: why is yarn a dev dependency?

File: package.json [hunk 2]
```diff
@@ -38,12 +38,16 @@
...
-    "bluebird": "^2.10.2",
-    "image-size": "^0.3.5",
-    "lodash": "^3.10.1",
-    "phantom": "^0.8.0",
-    "phantomjs": "^1.9.18",
-    "request-promise": "^0.4.3",
-    "yargs": "^3.26.0"
+    "bluebird": "^3.5.0",
+    "image-size": "^0.5.1",
+    "imagely": "imagely-v0.1.12.tgz",
```
Comment (mpetrovich) [inline package.json:43]: what's this about?

Comment (choshun) [reply package.json:43]: I mistakingly added that, but took it out in a later PR.

How I was trying to test imagely locally as a package using `yarn pack`, http://podefr.tumblr.com/post/30488475488/locally-test-your-npm-modules-without-publishing

File: package.json (hunk 1 above)
Comment (choshun) [reply package.json:51]: My thinking was because you need it to develop, but not to run the application. I'll defer to whatever you think is best.

Comment (mpetrovich) [reply package.json:51]: does this imply that it _won't_ work with `npm install`?

**Comment by mpetrovich** [comment]
These changes are included as part of https://github.com/mpetrovich/imagely/pull/4

**Pull request closed without merging**